velox/dwio/parquet/thrift/CMakeLists.txt

```diff
@@ -12,5 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-velox_add_library(velox_dwio_parquet_thrift ParquetThriftTypes.cpp)
-velox_link_libraries(velox_dwio_parquet_thrift arrow thrift Boost::headers fmt::fmt)
+include(FBThriftCppLibrary)
+
+add_fbthrift_cpp_library(velox_dwio_parquet_thrift_raw parquet.thrift)
+velox_add_library(velox_dwio_prquet_thrift)
+if(VELOX_MONO_LIBRARY)
+  target_link_libraries(velox velox_dwio_parquet_thrift_raw)
+else()
+  target_link_libraries(velox_dwio_parquet_thrift PUBLIC velox_dwio_parquet_thrift_raw)
+endif()
```

Comment:
We can cleanup this part once we cleanup `velox_add_library()` and its family.

Reply:
I assume you mean because `velox_link_libraries` drops the link to `velox..._raw`? You can add it [here](https://github.com/facebookincubator/velox/blob/main/CMake/VeloxUtils.cmake#L174) to override that behavior.

Reply:
I want to use `INTERFACE` here... (The latest code is changed from this.)